public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -1466,6 +1311,7 @@ export const Explorer = ({
         explorerData,
         http,
         query,
+        notifications,
```

Comment:
?

Reply:
The notification is used by log pattern after modularization, and for easy access to all components that needs to access notification,  I added to context

Reply:
Curious are there plans to expose CoreStart/CoreSetup services as context or js services instead of passing objects to functions? 

In code, objects like `http`, `pplService` are passed down everywhere. it's tedious and becomes messy, and sometimes it's not easy or not possible to pass the dependency in arguments (for example inside an embeddable component)

Reply:
Yes, it's in the plan, one to-do item on my end is to clear up services that are supposed to be exposed as context.